### DISCUSSION TITLE
Disc Numbers

### DISCUSSION THREAD

**Discussion by jaymarvels** [advplyr/audiobookshelf#934] [Q&A] [answered]
Currently imported an audio book which is split over discs, so disc 1-9, each in their own folder and within each disk (folder) is a number of tracks.

I have meta tagged them under discnumber as per docs but when I import them in ABS is seeing them as separate library entries.

E.g

![image](https://user-images.githubusercontent.com/7057631/186527844-048279fb-643f-4f3f-869f-57bfebc2baf0.png)

**Comment by advplyr** [comment, marked as answer]
You have to name each folder in the format of `CD1`, `CD01`, or `CD001`

Cd folders aren't really supported yet but I added this one case that will look for folders named in that way and use that number as the cd number.

**Reply by jaymarvels** [reply]
Perfect, thanks

**Comment by Guruleenyc** [comment]
Can you use series metadata instead of restructuring folders?

**Reply by advplyr** [reply]
Yes, see the documentation and use the "Prefer Audio Metadata" setting to override the folder names.
